### ISSUE TITLE
got an unexpected keyword argument 'service'

### ISSUE THREAD

**Issue by sachaz** [aau-zid/BigBlueButton-liveStreaming#166]
**Stream error: not enough frames to estimate rate; consider increasing probesize**

Actual packages version:
~~~
ii  bbb-apps-akka                        2.4.0-103                                   all          BigBlueButton Apps (Akka)
ii  bbb-config                           1:2.4.0-53                                  amd64        BigBlueButton configuration utilities
ii  bbb-etherpad                         1:2.4.0-14                                  amd64        The EtherPad Lite components for BigBlueButton
ii  bbb-freeswitch-core                  2:2.4.0-28                                  amd64        BigBlueButton build of FreeSWITCH
ii  bbb-freeswitch-sounds                1:1.6.7-5                                   amd64        FreeSWITCH Sounds
ii  bbb-fsesl-akka                       2.4.0-43                                    all          BigBlueButton FS-ESL (Akka)
ii  bbb-html5                            1:2.4.0-2561                                amd64        The HTML5 components for BigBlueButton
ii  bbb-learning-dashboard               1:2.4.0-29                                  amd64        BigBlueButton bbb-learning-dashboard
ii  bbb-mkclean                          1:0.8.7-4                                   amd64        Clean and optimize Matroska and WebM files
ii  bbb-playback                         1:2.4.0-19                                  amd64        BigBlueButton playback
ii  bbb-playback-presentation            1:2.4.0-9                                   amd64        BigBluebutton playback of presentation
ii  bbb-record-core                      1:2.4.0-14                                  amd64        BigBlueButton record and playback
ii  bbb-web                              1:2.4.0-99                                  amd64        BigBlueButton API
ii  bbb-webrtc-sfu                       1:2.4.0-65                                  amd64        BigBlueButton WebRTC SFU
ii  bigbluebutton                        1:2.4.0-2                                   amd64        Open source web conferencing platform (bbb)
~~~
~~~
google-chrome --product-version | grep -o "[^\.]*\.[^\.]*\.[^\.]*"
100.0.4896
chromedriver --version
ChromeDriver 100.0.4896.60 (6a5d10861ce8de5fce22564658033b43cb7de047-refs/branch-heads/4896@{#875})
~~~

When I'm trying to launch the stream I got this error:

~~~
INFO:root:Starting browser!!
Traceback (most recent call last):
  File "stream.py", line 364, in <module>
    set_up()
  File "stream.py", line 152, in set_up
    browser = webdriver.Chrome(service=Service('./chromedriver'), options=options)
TypeError: __init__() got an unexpected keyword argument 'service'
~~~

**Comment by sachaz** [comment]
corrected with selenium 4.0 & python 3.7

**Comment by Amanb1145** [comment]
> corrected with selenium 4.0 & python 3.7

Hey Can you share Dockerfile if possible how did you install selenium 4.0 and Pyhton 3.7
I have been trying it but getting issues.
Thanks
